tasks/blockchain/boot.yml

```diff
@@ -16,17 +17,17 @@
     image: "{{ bc_polkadot_image }}"
     command: |
       /usr/local/bin/polkadot -d /state
-      --chain=bench --validator --key //Alice --ws-external --rpc-external
+      --chain=bench --validator --alice --ws-external --rpc-external --rpc-cors=all
```

Comment:
We don't need "Set producer keys" here because of `--alice`, do we?

Reply:
Yes, if you provide the `--alice` flag, the node start with Alice keys injected